packages/client/src/client/App/Credit/CreditRfqs/CreditRfqCards/styled.tsx

```diff
@@ -76,11 +76,7 @@ export const NoRfqsWrapper = styled.div`
 
 // Card Footer
```

Comment:
started off taking against this pointless-looking comment .. 😂 
.. then found `AcceptedCardState` is not used, and the final item `QuantityTypography` is not part of the footer .. 
I used to have a mode set in VS Code where it annotated the usage count of each identifier .. got annoying, but might be useful when refactoring, like this ..?